frontend/components/ClubEditPage.tsx

```diff
@@ -350,6 +351,15 @@ const ClubForm = ({
           </>
         ),
       },
+      {
+        name: 'organization',
+        label: 'Club Management',
+        content: (
+          <>
+            <ClubManagementCard club={club} />
+          </>
+        ),
+      },
```

Comment:
One question I have is that Umbrella Orgs don't really exist on Hub@Penn as far as I can tell, do you think this would be worth only displaying on Penn Clubs? Lmk what your thoughts are, I can go either way on this one.

Reply:
Good point, I didn't bother disabling this on Hub@Penn but ideally this would be hidden on Hub@Penn.

Reply:
Disabled!

Reply:
Awesome thanks Eric!